suite-native/storage/src/migrations/account/v2.ts

```diff
@@ -0,0 +1,30 @@
+interface AccountMetadata {
+    accountLabel?: string;
+}
+
+type OldAccount = {
+    metadata: AccountMetadata;
+    [key: string]: any;
+};
+
+type MigratedAccount = {
+    accountLabel?: string;
+    [key: string]: any;
+};
+
+export const migrateAccountLabel = (oldAccounts: OldAccount[]): MigratedAccount[] =>
+    oldAccounts.map(oldAccount => {
+        if (!oldAccount.metadata || !oldAccount.metadata.accountLabel) {
+            return oldAccount;
+        }
+
+        const { accountLabel, ...metadataWithoutAccountLabel } = oldAccount.metadata;
+
+        return {
+            ...oldAccount,
+            metadata: {
+                ...metadataWithoutAccountLabel,
+                accountLabel,
+            },
+        };
```

Comment:
This actually changes nothing, right? 🤔  now I'm wondering why the migration would be needed